src/promptflow/promptflow/executor/_line_execution_process_pool.py

```diff
@@ -167,6 +166,7 @@ def start(self):
             "output_queues": self._output_queues,
             "process_info": process_info,
             "process_target_func": _process_wrapper,
+            "run_id": self._run_id,
```

Comment:
It may be enough to use index. There is no run_id in line process pool for executor server.

Reply:
fixed